subworkflows/local/prepare_genome.nf

```diff
@@ -38,14 +40,31 @@ workflow PREPARE_GENOME {
     ch_versions = ch_versions.mix(UNTAR_VEP_CACHE.out.versions)
 
     UNTAR_VEP_CACHE.out.untar
-        .map { meta, files -> [files] }
+        .map { meta, files -> [ files ] }
         .collect()
         .set { untarred_vep }
 
+    // Read and store paths in the vep_plugin_files file
+    if ( split_vep_files ) {
+        ch_vep_extra_files_unsplit
+            .splitCsv ( header:true )
+            .map { row ->
+                path = file(row.vep_files[0])
+                if(path.isFile() || path.isDirectory()){
```

Comment:
Would `path.exists()` not be sufficient (and clearer) here?

Reply:
Sounds good!